clusterMembership/clusterMembership.go

```diff
@@ -0,0 +1,21 @@
+package clusterMembership
+
+
+type Node interface {
+	GetId() string
+	SendMessage(msg string) error
```

Comment:
So the idea is you don't you just interact with nodes by sending them messages, via the Cluster Interface